app/src/androidTest/java/com/example/android/courtcounter/Tests/ExampleInstrumentedTest.java

```diff
@@ -1,4 +1,4 @@
-package com.example.android.courtcounter;
+package com.example.android.courtcounter.Tests;
```

Comment:
Probably my bad, but the package name should be in lower case, literally had Robert point this out to me in another PR like two days ago - https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html
So `com.example.android.courtcounter.tests;`